src/Components/Application/reviewReducer.js

```diff
@@ -22,14 +22,10 @@ import { produce } from "immer";
  */
 export const INSERT_COMMENT = "INSERT_COMMENT";
 export const UPDATE_RATING = "UPDATE_RATING";
-export const LOAD_REVIEW = "LOAD_REVIEW";
 
 const MASTER_ID = "master";
 
 export function reviewReducer(review, action) {
-  if (action.type === LOAD_REVIEW) {
-    return action.review;
-  }
```

Comment:
Just confirming this was meant to be removed. Do we not have this action anymore?

Reply:
Also wondering this. @xinhaoz

Reply:
Yeah the only use case has been deleted.